mysql-test/suite/rpl/t/rpl_bug85158.test

```diff
@@ -0,0 +1,108 @@
+#
+# Testcase for bug 85158.
+#
+--source include/not_group_replication_plugin.inc
+--source include/have_gtid.inc
+--source include/have_binlog_format_row.inc
+
+--let $rpl_multi_source= 1
+--let $rpl_topology= 1->2,2->3,1->3
+--source include/rpl_init.inc
+
+# mi writes threshold, if the difference between count of writes to
+# mi repository before and after the actions is greater than the
+# threshold, the test failed
+--let $threshold=5
+
+# The number of inserts on server_1, if the bug is not fixed the amount
+# of mi repository writes will be greater or equal to this value
+--let $inserts_number=50
+
+--echo ###
```

Comment:
Very redundant comment given the "STOP SLAVE" below

Reply:
I would not say so because this is not only the comment but also some mark in *.result file which shows the certain test step. *.result file becomes more readable with such marks.

Reply:
OK